xbmc/guilib/GUIFontTTF.cpp

```diff
@@ -805,7 +805,7 @@ CGUIFontTTF::Character* CGUIFontTTF::GetCharacter(character_t chr, FT_UInt glyph
   {
     if (m_char[i].letter < 255)
     {
-      character_t ch = ((m_char[i].glyphAndStyle & 0xffff0000) >> 8) | (m_char[i].glyphIndex);
+      character_t ch = ((m_char[i].glyphAndStyle & 0xffff0000) >> 8) | (m_char[i].letter);
```

Comment:
```suggestion
      character_t ch = ((m_char[i].glyphAndStyle & 0xffff0000) >> 8) | m_char[i].letter;
```
Parenthesis is not necessary here